data/transit/route/subway.json

```diff
@@ -3,10 +3,14 @@
     {
       "displayName": "BART",
       "id": "bart-907487",
-      "locationSet": {"include": ["001"]},
+      "locationSet": {"include": ["us"]},
       "tags": {
         "network": "BART",
-        "operator": "BART",
+        "network:wikidata": "Q610120",
+        "network:wikipedia": "en:Bay Area Rapid Transit",
+        "operator": "Bay Area Rapid Transit District",
```

Comment:
Interesting! Whatever you want to do is fine here.. I love that people are putting a lot more care and thought into transit tagging because of this.. 😁

Reply:
Like the existing brand stuff, it just makes sense, get it right once, get it right everywhere.